src/models/maze.c

```diff
@@ -41,9 +44,39 @@ bool retrieveMazeFromFile()
 
     resetGameMaze();
 
+    fillMazeWithCoins();
     return true;
 }
 
+void fillMazeWithCoins()
+{
+    for (int i = 0; i < MAP_HEIGHT; i++)
+    {
+        for (int j = 0; j < MAP_WIDTH; j++)
+        {
+            struct Position pos = getGridPosToUiPos((struct Position){j,i});
+            SDL_SetColorKey(pSurfacePacmanSpriteSheet, true, 0);
+
+            if (gameMaze[i][j] == SMALL_COIN)
+            {
+                SDL_Rect destinationRect = {pos.x+13, pos.y+13, 13, 13};
+                SDL_BlitScaled(pSurfacePacmanSpriteSheet, &imgMazeSmallCoin, pSurfaceWindow, &destinationRect);
+            } else if (gameMaze[i][j] == BIG_COIN) {
+                SDL_Rect destinationRect = {pos.x+8, pos.y+8, 20, 20};
```

Comment:
Maybe we can create a function to avoid duplicating this code part.

Also, we can create consts for this value (x+8, 20 ...)

Reply:
d93a846